01-node-tutorial/answers/.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
The leading `/` is not necessary here. The leading slash means you're ignoring node_modules in this exact folder, but not node_modules in nested folders. In reality, you don't need that level of specificity here.